pd/src/main.rs

```diff
@@ -57,9 +57,20 @@ enum RootCommand {
         /// Bind the gRPC server to this port.
         #[clap(short, long, default_value = "8080")]
         grpc_port: u16,
-        /// Bind the metrics endpoint to this port.
+        /// bind the metrics endpoint to this port.
         #[clap(short, long, default_value = "9000")]
         metrics_port: u16,
+        /// Proxy Tendermint requests against the gRPC server to this host.
+        #[clap(
+            short,
+            long,
+            default_value = "testnet.penumbra.zone",
+            parse(try_from_str = url::Host::parse)
+        )]
+        tendermint_host: url::Host,
```

Comment:
do we want this to default to `testnet.penumbra.zone`? since the `pd` app will be driven by a local tendermint instance, there will be one available, and it seems better to use that one.

if we make the default be `127.0.0.1`, then the Quickstart workflow will work without extra arguments.

or, do we actually want this to be two seperate arguments? what if it just took a `std::net::SocketAddr`?